Shared/Scripts/bxl.ps1

```diff
@@ -81,10 +81,9 @@ param(
     [Parameter(Mandatory=$false)]
     [switch]$DeployStandaloneTest = $false,
 
-    # Task 544796 to enable this
     [Parameter(Mandatory=$false)]
     [ValidateSet("Disable", "Consume", "ConsumeAndPublish")]
-    [string]$SharedCacheMode = "Disable",
+    [string]$SharedCacheMode = "Consume",

```

Comment:
> Do we have to do anything to reenable our dev cache publishing builds running in CloudBuild?

I don't think so, because I never disabled any publishing to the shared cache that is going on in CloudBuild.

Reply:
That said, only net472 bits are currently publishing to shared cache